DNN Platform/DotNetNuke.Web.Client/Cdf/CssMediaType.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Web.Client.Cdf
+{
+    /// <summary>
+    /// Specifies the CSS media types that can be used.
+    /// </summary>
+    public enum CssMediaType
+    {
+        /// <summary>
+        /// Applies to all media types.
+        /// </summary>
+        All,
```

Comment:
Is this something intended to be allowed within the `StylesheetResource` declaration?  If so I think it would be a great add

Reply:
This is temporary old code to avoid breaking changes